src/layers/normalise.jl

```diff
@@ -1,30 +1,28 @@
+# Utility function for applying LayerNorm before a block
+prenorm(planes, fn) = Chain(fn, LayerNorm(planes))
```

Comment:
If the block is `fn` layer norm is being applied afyer

Reply:
Whoops 😅